### DISCUSSION TITLE
折线图 scale 设置max之后，图表显示异常

### DISCUSSION THREAD

**Discussion by Dawnwangzi** [antvis/G2#4945] [Q&A] [answered]
codesandbox链接
https://codesandbox.io/s/000pzp?file=/index.ts

![image](https://user-images.githubusercontent.com/20938424/235042111-b89820c2-ddd3-4b99-b929-d66edb7a004c.png)
![image](https://user-images.githubusercontent.com/20938424/235042361-9243123d-0e55-46e8-911d-a2d3b1183536.png)

是什么原因导致设置max 有一定概率 图上什么都不显示？
在max下的数据其实有很多，但是就是不显示，只有hover上去才闪出来一个点

**Comment by Dawnwangzi** [comment]
![image](https://user-images.githubusercontent.com/20938424/235072158-2e9e683d-9206-4ea2-b3fa-ed69c0883308.png)
还有 设了min之后 min轴以下的图形怎么才能直接截断不显示？

**Comment by Dawnwangzi** [comment]
@pearmini 请问这个问题有解决办法吗

**Comment by hustcc** [comment, marked as answer]
- 设置 min 后，折线超出下方的问题，加上 limitInPlot 可以解决。

```ts
const chart = new Chart({
    container: 'container',
    autoFit: true,
    height: 500,
    limitInPlot: true,  // 👈🏻 这里
  });
```

- max 设置为 2000 不显示的问题，需要再排查，确实有点奇怪，不过对你这个数据量级差距大的情况，建议使用 log 类型

```ts
chart.scale({
    ts: {
      tickCount: 10,
      type: 'timeCat',
    },
    value: {
      nice: false,
      type: 'log',  // 👈🏻 这里
      max: 2000,
      // min: 1500
    }
  });
  
```
